oauthenticator/auth0.py

```diff
@@ -37,12 +37,24 @@
 from traitlets import Unicode
 
 from .oauth2 import OAuthenticator
+from .oauth2 import OAuthLogoutHandler
+
+
+class Auth0LogoutHandler(OAuthLogoutHandler):
+    async def render_logout_page(self):
+        if self.authenticator.logout_redirect_url:
+            self.redirect(self.authenticator.logout_redirect_url)
```

Comment:
Now that the `logout_redirect_url` config is on the base class, this redirect behavior should probably be on the base OAuthLogoutHandler class to match, and Auth0 doesn't need a custom `logout_handler` class anymore.

Reply:
Thanks, @minrk! I just moved the redirect behavior and rebased the PR